lib/Doctrine/ORM/Query/ParserResult.php

```diff
@@ -20,6 +22,7 @@ class ParserResult
         'sqlExecutor' => '_sqlExecutor',
         'resultSetMapping' => '_resultSetMapping',
         'parameterMappings' => '_parameterMappings',
+        'sqlFinalizer' => 'sqlFinalizer',
```

Comment:
This seems uneeded

Reply:
I'll need to check. I had the impression that without it, this would be missing from the serialization result

Reply:
`ParserResultSerializationTest` shows it is indeed needed, otherwise the `SqlFinalizer` is lost

Reply:
Oh right, I was mislead by the naming